packages/fast-components-styles-msft/src/breadcrumb/index.ts

```diff
@@ -29,7 +29,7 @@ const styles: ComponentStyles<BreadcrumbClassNameContract, DesignSystem> = (
             ...typographyPattern.rest,

```

Comment:
are `designSystem` and `direction` variables above being used? if not, can you remove them?

Reply:
yes for applyLocalizedProperty